README.md

```diff
@@ -137,6 +137,8 @@ yarn run build:development
 
 Also, when running `yarn install`, copied builds will be overwritten and will require running `yarn run build-consumer PROJECT_DIRECTORY` again.
 
+If you encounter compilation errors in your consuming project, ensure the dependencies in the consuming project match the versions in `polaris-react`.
```

Comment:
Is this supposed to be "peer dependencies" instead of "dependencies"?

Reply:
`@shopify/polaris-tokens` is not listed in the `peerDependencies` in package.json so I chose to use dependencies.